contracts/HybridCustody.cdc

```diff
@@ -626,11 +637,10 @@ pub contract HybridCustody {
         configured for the provided parent address. Once done, the parent will not have any valid capabilities
         with which to access the child account.
         */
-        pub fun removeParent(parent: Address): Bool {
+        access(contract) fun removeParent(parent: Address): Bool {
```

Comment:
I think this should be pub. Making this `access(contract)` means feels like it's trying to prohibit what we can't prevent.

Whoever has direct access to this resource (or `ChildAccountPrivate` Capability) likely also has access to the underlying account, so they could unlink any of the Capabilities a parent account relies on. By unlinking those Capabilities, one would in essence be removing the parent account, bypassing a restriction we're perceptually attempting to enforce by making this state change `access(contract)`.

Open for discussion on this, of course. Any objections to making this `pub`?

Reply:
Agreed this should be public. The rules as I see them are:

1. The parent can remove a child at any time
2. The owner of the child account can remove any parent at any time

@web3ashlee let's sync tomorrow and go through how we want to structure this

Reply:
Great callout @sisyphusSmiling. Agreed and fixed.